docs/pages/2019_MARVEL_Psik_MaX/sections/bands.rst

```diff
@@ -1,46 +1,55 @@
+.. _bands:
+
 A real-world WorkChain: computing a band structure
 ==================================================
 
-**Note**: *If you still have enough time, you might want to check first
-Appendix [sec:convpressure] before continuing with this section.*
+**Note:** *If you still have enough time, you might want to check first*
```

Comment:
by the way, RST has the much nicer

.. note::

   ...

Reply:
Thanks, done.